client/my-sites/sidebar/index.jsx

```diff
@@ -230,7 +232,9 @@ export class MySitesSidebar extends Component {
 				tipTarget="earn"
 				expandSection={ this.expandToolsSection }
 			>
-				<Badge type="info">{ translate( 'New' ) }</Badge>
+				{ moment( userDate ).diff( moment( '2019-11-26 00:00:00' ) ) < 0 && (
```

Comment:
Same thing here. I'd prefer to use `isAfter` or `isBefore` to make the check clearer. Thanks!